airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -2377,6 +2377,7 @@
     hosts:
       - "${subdomain}.zendesk.com"
       - zendesk.com
+  maxSecondsBetweenMessages: 7200
```

Comment:
For my own edification, where do we define the default 1hr?

Reply:
in a feature flag: https://app.launchdarkly.com/default/production/features/heartbeat-max-seconds-between-messages/targeting this one